RebelAllianceBank/Classes/Customer.cs

```diff
@@ -99,5 +99,140 @@ public void CreateAccount()
                 }
             } while (createAccount == false);
         }
+
+        public void TakeLoan()
```

Comment:
Kanske kan lägga till en while loop så om man skriver in ett lån över sitt maximal lån så kan man försöka igen och då skriva en mindre summa?

Reply:
Löste det genom att kalla på Metoden igen :)
![image](https://github.com/user-attachments/assets/79ea4516-eaa9-4743-8bc6-d1995100b0af)